.github/workflows/release.yml

```diff
@@ -46,13 +46,27 @@ jobs:
         git config --local user.name "${GITHUB_ACTOR}"
 
     - name: Build & Publish
+      id: build
       run: |
         echo Version level: ${{ github.event.inputs.versionLevel }}
         npm ci
         npm run build --if-present
-        npm version ${{ github.event.inputs.versionLevel }} --message "${{ github.event.inputs.commitMessage }}"
+        VERSION=$(npm version ${{ github.event.inputs.versionLevel }} --message "${{ github.event.inputs.commitMessage }}")
+        VERSION=${VERSION:1} # remove 'v' prefix
+        echo "##[set-output name=version;]$VERSION"
         git push --follow-tags
         npm config set //registry.npmjs.org/:_authToken=$NODE_AUTH_TOKEN
         npm publish --access public
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+    - name: Publish Docker image to Docker Hub
+      env:
+        DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
```

Comment:
`DOCKER_TOKEN` needs to be changed to `secrets.DOCKERHUB_TOKEN`.
And a new `DOCKERHUB_USERNAME` needs to be created and set to `secrets. DOCKERHUB_USERNAME `.

Reply:
Done!